### ISSUE TITLE
Auth activity crash with v 3.1.2

### ISSUE THREAD

**Issue by bazimogmbh** [dropbox/dropbox-sdk-java#267]
Calling  `Auth.startOAuth2Authentication(context, DROPBOX_KEY)` causes a crash, **if the dropbox app is not installed** on the device. It was working perfectly in previous version.

```
java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String com.dropbox.core.TokenAccessType.toString()' on a null object reference
        at com.dropbox.core.android.AuthActivity.createExtraQueryParams(AuthActivity.java:707)
        at com.dropbox.core.android.AuthActivity.startWebAuth(AuthActivity.java:674)
        at com.dropbox.core.android.AuthActivity.access$100(AuthActivity.java:41)
        at com.dropbox.core.android.AuthActivity$3.run(AuthActivity.java:550)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:6986)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:494)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1445)
```

**Comment by greg-db** [comment]
Thanks for the report! We're looking into it.

**Comment by adityabhaskar** [comment]
Same issue. Rolling back to previous version.

**Comment by jiuyangzhao** [comment]
Released 3.1.3 to fix that
